.github/workflows/README.md

```diff
@@ -119,6 +119,8 @@ This job is responsible for deploying to our production, cloud kubernetes enviro
 
 ## Pro
 
+| **NOTE**: When developing for both pro / budibase repositories, your branch names need to match, or else the correct pro image doesn't get run within your CI job.
```

Comment:
Would remove the mention of `image` here as there are no images involved, pro is just a library of source code

Reply:
Sure! thanks for that